lib/boombox.ex

```diff
@@ -101,13 +102,32 @@ defmodule Boombox do
 
   If the input is `{:stream, opts}`, a `Stream` or other `Enumerable` is expected
   as the first argument.
+
+  If `:enforce_transcoding?` options is set to `true`, boombox will perform audio and/or video
+  transcoding, even if it is not necessary. By default this option is set to `false`.
+
+  ```
+  Boombox.run(input: "path/to/file.mp4", output: {:webrtc, "ws://0.0.0.0:1234"}, enforce_transcoding?: true)
+  ```
   """
-  @spec run(Enumerable.t() | nil, input: input(), output: output()) :: :ok | Enumerable.t()
+  @spec run(Enumerable.t() | nil,
+          input: input(),
+          output: output(),
+          enforce_transcoding?: boolean()
```

Comment:
I think we should make it possible to enforce transcoding of only audio or video and also specify the desired output codec